config/puma.rb

```diff
@@ -9,7 +9,7 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 3000 }
+port        ENV.fetch("PORT") { 3000 }, "0.0.0.0"
```

Comment:
これをRails本体で指定すべきかは少し迷いますね……。

「Dockerから開発するときは `rails s -b 0.0.0.0` ってしてね！」って書いて、puma.rb自体で指定するのは避けるべきな気もします。

Reply:
それなら環境変数から指定するようにするのはどうですかね
docker内ではオプションを省略できるようにした上で、通常は今までと同じ動作にできるはずです

```suggestion
port        ENV.fetch("PORT") { 3000 }, ENV["HOST"]
```

Reply:
動くの確認できました！